examples/gno.land/p/demo/grc/grc1155/basic_grc1155_token_test.gno

```diff
@@ -35,8 +35,8 @@ func TestBalanceOf(t *testing.T) {
 		t.Errorf("should not be nil")
 	}
 
-	addr1 := users.AddressOrName("g1var589z07ppjsjd24ukm4uguzwdt0tw7g47cgm").Resolve()
-	addr2 := users.AddressOrName("g1us8428u2a5satrlxzagqqa5m6vmuze025anjlj").Resolve()
+	addr1 := std.Address(users.AddressOrName("g1var589z07ppjsjd24ukm4uguzwdt0tw7g47cgm"))
```

Comment:
Is it necessary to still include the `users.AddressOrName` cast if it isn't calling `Resolve`?  There are lots of other places that still do this as well.

Reply:
Fixed: 9ee009bffb21fb419c17b424719f6c1c98f946d3